astrowidgets/core.py

```diff
@@ -545,11 +549,19 @@ def get_markers(self, x_colname='x', y_colname='y',
 
             return stacked
 
+        # We should always allow the default name. The case
+        # where that table is empty will be handled in a moment.
+        if (marker_name not in self._marktags and
+                marker_name != self._default_mark_tag_name):
+            raise ValueError(f"No markers named '{marker_name}' found.")
+
         try:
             c_mark = self._viewer.canvas.get_object_by_tag(marker_name)
-        except Exception as e:  # No markers
-            self.logger.warning(str(e))
-            return
+        except Exception:
+            # No markers in this table. Issue a warning and continue
+            warnings.warn(f"Marker set named '{marker_name}' is empty",
+                          category=RuntimeWarning)
```

Comment:
Is `RuntimeWarning` the best one to use? According to https://docs.python.org/3/library/warnings.html#warning-categories , it is for "warnings about dubious runtime features."

Reply:
I wasn't sure which one to go with....I picked runtime because that is what numpy does for things like underflow/overflow errors. Could change it to `UserWarning`.

Reply:
Empty marker list isn't quite the same as under/overflow. I would prefer `UserWarning` but I'll leave the final decision to you.